types/gapi.auth2/index.d.ts

```diff
@@ -48,6 +48,7 @@ declare namespace gapi.auth2 {
      */
     grantOfflineAccess(options: {
       scope?: string;
+      prompt?: string;
```

Comment:
Consider switching this to `"select_account" | "consent"`.

How come this type is has `redirect_uri` (which isn't documented) but not `app_package_name`?